cloud/gsutil/clone.sh

```diff
@@ -0,0 +1,10 @@
+DONT_RUN_THIS
```

Comment:
lol how about exit the script unless a -force param is set, or something like that